### DISCUSSION TITLE
Examples of merging audio files

### DISCUSSION THREAD

**Discussion by ra0x3** [zmwangx/rust-ffmpeg#73] [Q&A]
I have two files `audio1.mp3` and `audio2.mp3`. 

I would like to merge these two files to produce` audio3.mp3`, with `audio3.mp3` being almost like a bitwise AND operator over the two input files. I've read a [really great Stackoverflow thread about doing this with ffmpeg here](https://stackoverflow.com/questions/14498539/how-to-overlay-downmix-two-audio-files-using-ffmpeg). 

So my question is, could anyone possibly give a tiny example on how to do this using rust-ffmpeg.

Cheers 🎉

**Comment by thewh1teagle** [comment]
Hey @ra0x3 
Did you found solution?

I'm trying to mimic the following command for merging audio files:

```console
ffmpeg -i a.wav -i b.wav -filter_complex amix=inputs=2:duration=shortest -ac 2 merged.wav
```
